src/components/ConversionProcessingOverlay/style.css

```diff
@@ -10,53 +10,46 @@
     justify-content: center;
 }
 
-.container h2,p,h3 {
-  color: var(--dark);
+.container h2,
+.container h3 {
```

Comment:
if you don't have the parent group after the `,` it will style every `<p>` as with the dark color